test/platform/test-platform-execute.ts

```diff
@@ -33,33 +46,23 @@ async function registerDevice(
   Promise.resolve();
 }
 
-function createSimpleExecuteCommands(
```

Comment:
is this related to this change (if yes, maybe update the description to list it)

Reply:
This was from #39 and was merged.  Should be fixed now.